### PR TITLE
Add Brazilian Porguguese translation

### PR DESCRIPTION
**Pull request by DanielTolentino**
Hello, this is the Brazilian Portuguese translation. Thanks for the app!

### CODE REVIEW

File: po/extra/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+msgid "Support multitouch gestures (thanks to @JeysonFlores)"
+msgstr "Suporte à gestos multitoques (agradecimentos a @JeysonFlores)"
+
+#: data/pinit.appdata.xml.in:108
+msgid "Fix unreadable description text in AppCenter"
+msgstr "Corrige texto de descrição ilegível no AppCenter"
+
+#: data/pinit.appdata.xml.in:109
+msgid "Save and restore window position"
+msgstr ""
```
Comment (ryonakano) [inline po/extra/pt_BR.po:121]: Here

File: po/extra/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+#: data/pinit.appdata.xml.in:113
+msgid "Add Spanish translation (thanks to @JeysonFlores)"
+msgstr ""
```
Comment (ryonakano) [inline po/extra/pt_BR.po:125]: Here

File: po/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+"PO-Revision-Date: 2021-11-29 16:05-0300\n"
+"Last-Translator: Daniel Tolentino Ferreira Santos <danieltolentino@protonmail.com>\n"
+"Language-Team: none\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+
```
Comment (ryonakano) [inline po/pt_BR.po:21]:
```suggestion

```

Seems like an extra new line here

**Review by ryonakano — COMMENTED** [review]
Hi, thank you for translating this app!

Would it be possible to translate the following missing translations? Otherwise really looks good!

File: po/extra/pt_BR.po (hunk 1 above)
Comment (DanielTolentino) [reply po/extra/pt_BR.po]: Oh, sorry, missed that!

File: po/extra/pt_BR.po (hunk 2 above)
Comment (DanielTolentino) [reply po/extra/pt_BR.po]: Done!

**Review by ryonakano — APPROVED** [review]
Perfect! Thank you :heart:

**Pull request merged**